tools/system_libs.py

```diff
@@ -1867,6 +1867,16 @@ def can_use(self):
     return settings.WASMFS
 
 
+class libwasmfs_no_fs(Library):
+  name = 'libwasmfs_no_fs'
+
+  def get_files(self):
+    return [utils.path_from_root('system/lib/wasmfs/no_fs.c')]
```

Comment:
Rather than overriding this method I think you can just set the set_files and srd_dir attributes.

Reply:
Thanks, done.